styles.css

```diff
@@ -1,20 +1,148 @@
-.home{
-position: relative;
-width: 375px;
-height: 6968px;
-background: #FFFFFF;
-}
-
-.cover{
-display: flex;
-flex-direction: column;
-align-items: flex-start;
-padding: 120px 24px;
-gap: 12px;
-position: absolute;
-width: 375px;
-height: 590px;
-left: 0px;
-top: 92px;
-background: #1C1A19;
-}
\ No newline at end of file
+
+.home {
+  /* width: 375px; */
+
+  /* height: 6968px; */
+  background: #fff;
+}
+
+.cover {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 591px;
```

Comment:
- Kindly it is not recommended to put a fixed size on your containers, they must be able to adapt to the size of the device.

Reply:
i erase that